packages/@coorpacademy-components/src/organism/review-no-skills/style.css

```
@@ -46,6 +46,13 @@
   width: 361px;
 }
 
+.imgNoSkillsReverse {
+  height: 361px;
```

suggestion to avoid repetition
here you can use the imgNoSkills properties this way, and add yours modified properties after:
```suggestion
  composes: imgNoSkills;
  margin-top: 0px;
  margin-bottom: 8px;
```